pkg/products/monitoring/dashboards/croResources.go

```diff
@@ -0,0 +1,1349 @@
+          "timeShift": null,
+          "title": "Postgres Storage",
+          "transformations": [
+            {
+              "id": "seriesToColumns",
+              "options": {
+                "byField": "productName"
+              }
+            },
+            {
+              "id": "organize",
+              "options": {
+                "excludeByName": {
+                  "Time": true,
+                  "__name__": true,
+                  "clusterID": true,
+                  "endpoint": true,
+                  "exported_namespace": true,
+                  "instance": true,
+                  "instanceID": true,
+                  "job": true,
+                  "namespace": true,
+                  "pod": true,
+                  "resourceID": true,
+                  "service": true,
+                  "strategy": true
+                },
+                "indexByName": {
+                  "Time": 1,
+                  "Value #A": 15,
+                  "Value #B": 14,
+                  "__name__": 2,
+                  "clusterID": 3,
+                  "endpoint": 4,
+                  "exported_namespace": 5,
+                  "instance": 6,
+                  "instanceID": 7,
+                  "job": 8,
+                  "namespace": 9,
+                  "pod": 10,
+                  "productName": 0,
+                  "resourceID": 11,
+                  "service": 12,
+                  "strategy": 13
+                },
+                "renameByName": {
+                  "Value #A": "Allocated",
+                  "Value #B": "Free",
+                  "productName": "Product"
+                }
+              }
+            },
+            {
+              "id": "calculateField",
+              "options": {
+                "mode": "reduceRow",
+                "reduce": {
+                  "reducer": "diff"
+                }
+              }
+            },
+            {
+              "id": "organize",
+              "options": {
+                "excludeByName": {},
+                "indexByName": {},
+                "renameByName": {
+                  "Difference": "Used"
+                }
+              }
+            }
+          ],
+          "type": "table"
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "Prometheus",
+          "fieldConfig": {
+            "defaults": {
+              "custom": {
+                "align": null
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              },
+              "unit": "decbytes"
```

Comment:
We should be consistent with the units used to avoid confusion (MB vs MiB in different places).
Please use `bytes` unit for all memory representations.